outputs.tf

```diff
@@ -42,6 +42,12 @@ output "bucket_storage_class" {
   value       = local.bucket_storage_class
 }
 
+output "bucket_resource_key" {
```

Comment:
resource key is bound to an instance, not a bucket right?

Reply:
Yes. The module generates one resource key per bucket. Each bucket defines the COS instance it belongs to, so there may be more than one instance and more than one key.
Resource keys have never been exposed, so if we are going to change this, then now would be the time.
We could only output a resource key when an instance is created. I would have to explore how this impacts the buckets module. Dropping this output.